### DISCUSSION TITLE
Shadow Tux entity

### DISCUSSION THREAD

**Discussion by AndrePinheiroPT** [SuperTux/supertux#2604] [Ideas]
(sorry for my bad english)
Hello, the shadow tux isn't my ideia, but the game should have a boss that chases the tux according to its movements, like Badeline fight at Celeste game, its interesting have a shadow tux like your dark side, for make it, you can record the moviments that tux does... Remenber, thats not my idea, I think the idea as created by @weluvgoatz ...

**Comment by Ordoviz** [comment]
Great idea! We need to decide what should happen if Tux uses an unstable tile. Will the shadow clone fall into his doom, Tux having outsmarted his shadow, or will the shadow jump mid-air (in this case he would be merely a ghost).

But I do not think that the shadow should be a boss. It would be more useful in an escape section.

See also [MarioWiki][1] and [Celeste Chapter 2 Playthrough][2] (7:36 to 10:08)

[1]: https://www.mariowiki.com/Cosmic_Clone
[2]: https://invidio.us/watch?v=VwlfUwLTEwI&t=456

**Comment by Rusty-Box** [comment]
@Ordoviz it was never meant as a boss AndrePinheiroPT only labeled it wrong. The overall idea from weluvgoatz was based on celeste.

**Comment by charla-con-galo** [comment]
good idea.

**Comment by Paulo-Tux** [comment]
Eu estive a fazer copyright de um sprite do Super Tux e modificei esse sprite. Estive a comparar o original com o modificado, ou seja estive a comparar o Tux com o Shadow Tux. Aqui mando os dois sprites: o original e o modificado. (I've been copyrighting a Super Tux sprite and I've modified that sprite. I've been comparing the original with the modified one, that is, I've been comparing Tux with Shadow Tux. Here I send the two sprites: the original and the modified one.)
![tux_stand-0](https://user-images.githubusercontent.com/126248752/228072668-e307b528-b159-40c6-9ca4-243bf007391b.png)
![shadow_tux_stand-0](https://user-images.githubusercontent.com/126248752/228072671-92b228d5-58b7-4967-9008-1cf3fee9bdb3.png)
